flowdb/bin/build/0020_schema_infrastructure.sql

```diff
@@ -175,3 +175,43 @@ CREATE SCHEMA IF NOT EXISTS infrastructure;
         hnd_type TEXT
 
         );
+
+    -- Table to record each time the cell info is updated
+    CREATE TABLE IF NOT EXISTS infrastructure.cells_table_versions (
+        id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+        ingested_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+        source_filename TEXT,
```

Comment:
Just wondering if it would be helpful to record a hash of the file or similar as well?

Reply:
And as discussed, file_date as well